modules/RStudio/lab/RStudio_Lab.Rmd

```diff
@@ -30,7 +47,8 @@ avg <- avg %>%
   filter(!is.na(number))
 ```
 
-When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. 
+When you click the **Knit** butto (at the top of RStudio), a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. 
+Next we will show you some code - no worries about understanding it yet, we just want to get you familiar with where code goes. When it is in an RMarkdown file chunk it is code that is saved to a file. When it is in the console, it is not saved. The console is useful for installing packages like we just did, this is because sometimes we have to interact with R directly and answer questions. Another reason that it is a good idea to install packages in the console is we only need to do it once, so we don't usually need to save the code.
```

Comment:
```suggestion
Next we will show you some code - no worries about understanding it yet, we just want to get you familiar with where code goes. 

When code is in an RMarkdown file chunk, it is saved to a file. When it is in the console, it is not saved. 

The console is useful for installing packages like we just did, this is because sometimes we have to interact with R directly and answer questions. Another reason that it is a good idea to install packages in the console is we only need to do it once, so we don't usually need to save the code.
```
>  this is because sometimes we have to interact with R directly and answer questions.
This is kind of vague, not sure students will understand